src/test/java/en/builin/CounterBitSetImplTest.java

```diff
@@ -0,0 +1,28 @@
+package en.builin;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CounterBitSetImplTest {
+
+    @Test
+    void properCounts() {
+
+        Counter counter = new CounterBitSetImpl(new IpToIntConverter());
+
+        assertEquals(0, counter.getResult());
+
+        counter.count("");
+        assertEquals(0, counter.getResult());
+
+        counter.count("122.163.242.115");
```

Comment:
а как насчет иных кейсов, негативных например?
```suggestion
        counter.count(" ");
        counter.count("555.555.555.555");
```

Reply:
для обеспечения производительности я допустил, что в файле все адреса соответствуют формату, т.к. проверка формата на миллионах строк съест прям порядочно.
добавил негативные тесты на неверный формат (ожидается, что будет исключение)